docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -77,70 +85,107 @@ feedbackIntegration({
+| `--z-index`     | `100000`                                           | The z-index of the widget                                                         |
+| `--inset`       | `auto 0 0 auto`                                    | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--page-margin` | `16px`                                             | How far from the edge of the screen should the widget be positioned.              |
 
-Colors can be customized via the Feedback class constructor or by defining CSS variables on the injected button. If you use the default button it will have a property `id="sentry-feedback`, meaning you can use the `#sentry-feedback` selector to define CSS variables to override.
+Colors can be customized by passing `themeLight` and/or `themeDark` to `feedbackIntegration({})`, or by defining CSS variables that override the default values.
 
 | key                       | css variable                   | light                                     | dark                                      | description                                                                                           |
 | ------------------------- | ------------------------------ | ----------------------------------------- | ----------------------------------------- | ----------------------------------------------------------------------------------------------------- |
-| `background`              | `--background`                 | `#ffffff`                                 | `#29232f`                                 | Background color of the widget (injected button and form)                                             |
-| `backgroundHover`         | `--background-hover`           | `#f6f6f7`                                 | `#352f3b`                                 | Background color of the injected button when in a hover state                                         |
 | `foreground`              | `--foreground`                 | `#2b2233`                                 | `#ebe6ef`                                 | Foreground color, e.g. text color                                                                     |
-| `error`                   | `--error`                      | `#df3338`                                 | `#f55459`                                 | Color used for error related components (e.g. text color when there was an error submitting feedback) |
-| `success`                 | `--success`                    | `#268d75`                                 | `#2da98c`                                 | Color used for success-related components (e.g. text color when feedback is submitted successfully)   |
+| `successForeground`       | `--success-foreground`         | `#268d75`                                 | `#2da98c`                                 | Color used for success-related components (e.g. text color when feedback is submitted successfully)   |
```

Comment:
```suggestion
| `successForeground`       | `--success-foreground`         | `#268d75`                                 | `#2da98c`                                 | Color used for success-related components (such as text color when feedback is submitted successfully).   |
```
We try to avoid latin abbreviations in docs